src/components/rack/rack.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Tile from '../tile';
+import './rack.css';
+
+export function Rack({id, letters, points}) {
+  return (
+    <div className="rack">
+      <div className="player">
+        {'Player ' + id}
+      </div>
+      <div className="score">
+        {'Score: ' + points}
+      </div>
+      <div className="tiles">
+        {letters.map((l, index) => <Tile key={index} id={`${id}_${index}`} letter={l} />)}
```

Comment:
do you ever use the index in id?  please name it `playerId` because it looks like it's a tile id.